test/FastTests/Corax/IndexSearcher.cs

```diff
@@ -499,7 +501,8 @@ public void SimpleInStatement()
         }
 
```

Comment:
https://issues.hibernatingrhinos.com/issue/RavenDB-17762
Please test this.

Reply:
That was passes, forgot to add it. I had the entire thing on the Mapping branch and had to transplant it. Got removed by mistake on the rebase.